### PR TITLE
Bumps rancher/wrangler to it's tagged v2 version.

### PR DESCRIPTION
**Pull request by KevinJoiner**
## Issue: https://github.com/rancher/rancher/issues/43618
The previous wrangler commit included all of the v2 changes. Except for the import path changes.

### CODE REVIEW

File: go.mod
Comment (KevinJoiner) [inline go.mod:7]: Remove and update version after https://github.com/rancher/dynamiclistener/pull/88 is merged

**Review by MbolotSuse — COMMENTED** [review]
LGTM Once the API Server and DL forks are replaced. One note, this bump for dynamic listener will also pull in https://github.com/rancher/dynamiclistener/commit/0a2d8dff62fde6dfaf009141c12b30c316d8cff3 . I don't think that should cause any issues, but make sure to give that a look before merging.

**Review by MbolotSuse — APPROVED** [review]
LGTM now that the fork has been replaced.

**Pull request merged**
